source/fx-guidelines/markup.md

```diff
@@ -0,0 +1,13 @@
+# Markup Guidelines
+___
+
+## Bootstrap Classes
+
+Since we’re using bootstrap, try to utilize bootstrap classes as much as possible rather than creating new styles or using inline styles.
+* <http://getbootstrap.com/css/>
+
+## Custom Styles
+
+We’re using BEM methodology and would advise everyone to do so when creating new classes or ids. There’s a lot of places in our app that currently don’t follow BEM but we’re trying to improve the codebase.
+* <https://en.bem.info/method/>
+* <http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/>
```

Comment:
Propose linking using markdown with something like: "[Here](http://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/) is a brief overview of BEM syntax.